spec/rails_helper.rb

```diff
@@ -71,3 +71,30 @@
   config.include Capybara::DSL
   config.include FactoryBot::Syntax::Methods
 end
+
+# Setup chrome headless driver
+Capybara.server = :puma, { Silent: true }
+
+Capybara.register_driver :chrome_headless do |app|
+  options = ::Selenium::WebDriver::Chrome::Options.new
+
+  options.add_argument("--headless")
+  options.add_argument("--no-sandbox")
```

Comment:
Don't worry about it for this PR, but just fyi it should be possible to run in the sandbox. Might need to switch to our govuk-ruby-images though (hence definitely not worth holding up this PR).